javalib/src/main/scala/java/text/DecimalFormat.scala

```diff
@@ -0,0 +1,673 @@
+package java.text
+
+import java.math.BigDecimal
+import java.math.BigInteger
+import java.math.MathContext
+import java.math.RoundingMode
+import java.util.Locale
+
+class DecimalFormat extends NumberFormat {
```

Comment:
Every file which was ported from Harmony needs that somewhere high up in the start of the file.